### PR TITLE
Be feature friend api nhan

### PR DESCRIPTION
**Pull request by nhtnhan**
Added follower api

### CODE REVIEW

**Review by fujishig2 — COMMENTED** [review]
Any reason why the gitignore is no longer looking for venv files? 

Also, the migrations folder is being added in this merge. Other than that lgtm

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,135 @@
...
+celerybeat.pid
+
+# SageMath parsed files
+*.sage.py
+
+# Environments
+.env
+.venv
+env/
+venv/
```
Comment (nhtnhan) [inline .gitignore:114]: @fujishig2 it is included here

Comment (fujishig2) [reply .gitignore:114]: Great! Must have missed it

**Review by fujishig2 — APPROVED** [review]
Looks good to me!

**Pull request merged**